install.sh

```diff
@@ -209,14 +209,12 @@ for package in $BOOTSTRAP_PACKAGES; do
 
 done
 
-# Create the device.json file if it doesn't exist.
-if [ ! -f "${CREW_CONFIG_PATH}/device.json" ]; then
-  cd "${CREW_CONFIG_PATH}"
-  echo_info "\nCreating new device.json."
-  jq --arg key0 'architecture' --arg value0 "${ARCH}" \
-    --arg key1 'installed_packages' \
-    '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
-fi
+# Create the device.json file.
```

Comment:
We want to be able to do a repair install, which only updates the device.json. why break that functionality?

Reply:
We don't have any files in `CREW_PREFIX` by this stage of the installer, so there's no point checking for a `device.json` as we know there will not be one.

Reply:
That's not true if you're doing a reinstall...

Reply:
@uberhacker are we recapitulating a previous discussion here?

Reply:
We clear out all of `CREW_PREFIX` as part of the installation process. See [here](https://github.com/chromebrew/chromebrew/blob/master/install.sh#L50-L58).

I'm happy to discuss *whether* we should be doing that at a later date, but this PR is not the place to discuss it. 

All I'm doing here is removing a redundant check.